app/api/v2/entities/user.rb

```diff
@@ -6,6 +6,7 @@ module Entities
       # Basic user info
       class User < API::V2::Entities::Base
         expose :email, documentation: { type: 'String' }
+        expose :nickname, documentation: { type: 'String' }
```

Comment:
this field should be added also on admin entities and management API entities

Reply:
I think the nickname filed already exposed in admin/mannagement API entities because we use `expose :user, using: API::V2::Entities::User`

Reply:
true